examples/outliers/alibi-detect-combiner/pipeline/combiner/Combiner.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import json
+import numpy as np
+
+class Combiner(object):
+
+    def aggregate(self, X, features_names=[]):
+        logging.warning(X)
```

Comment:
Normally it's useful to log input and ouput, but just as a heads up Clive just added functionality that allows you to autmoatically log input and output for each of the components in the graph automatically which could be leveraged instead of explicitly using this. Although this is not going to be available beyond the SNAPSHOT until 1.1, so more of a heads up

Reply:
Thanks, I will add it back there. I like that it will happen automatically in future - that sounds much better.

Reply:
Fixed in https://github.com/SeldonIO/seldon-core/pull/1306/commits/d9801b97d36d8e5def66b42db2467e996a185b26/